microsoft-365/managed-desktop/get-started/register-devices-partner.md

```diff
@@ -17,69 +17,41 @@ This topic describes the steps for Partners to follow to register devices. The p
 
 
 ## Prepare for registration 
-Before completing registration for a customer, you must first establish a relationship with them at the [Partner Center](https://partner.microsoft.com/dashboard). Be sure to select **Include delegated administration privileges for Azure Active Directory and Office 365**. Learn more at [Partner Center help](https://docs.microsoft.com/partner-center/request-a-relationship-with-a-customer).
+Before completing registration for a customer, you must first establish a relationship with them at the [Partner Center] (https://partner.microsoft.com/dashboard). See the [consent documentation] (https://docs.microsoft.com/windows/deployment/windows-autopilot/registration-auth#csp-authorization) for more details on that process. Any CSP partner can add devices on behalf of any customer, as long as the customer consents. You can also learn more about partner relationship and Autopilot permissions at [Partner Center help] (https://docs.microsoft.com/partner-center/customers_revoke_admin_privileges#windows-autopilot)
```

Comment:
```suggestion
Before completing registration for a customer, you must first establish a relationship with them at the [Partner Center] (https://partner.microsoft.com/dashboard). See the [consent documentation] (https://docs.microsoft.com/windows/deployment/windows-autopilot/registration-auth#csp-authorization) for more details on that process. Any CSP partner can add devices on behalf of any customer, as long as the customer consents. You can also learn more about partner relationships and Autopilot permissions at [Partner Center help] (https://docs.microsoft.com/partner-center/customers_revoke_admin_privileges#windows-autopilot).
```

Reply:
Please change "relationship" to "relationships" and add a final period (.) as indicated.